msg/position_controller_status.msg

```diff
@@ -14,3 +14,5 @@ float32 acceptance_radius		# the optimal distance to a waypoint to switch to the
 float32 yaw_acceptance			# NaN if not set
 
 float32 altitude_acceptance		# the optimal vertical distance to a waypoint to switch to the next
+
+bool launch_detection_running           # true if the launch detection is running
```

Comment:
Thought for later, it might be worth adding all launch detection states here for logging. That would be better than dumping the text messages into the log.

Reply:
Good Idea